feat: create come-si-realizza-design-servizi-pubblici (#1149)
italia/developers.italia.it: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md

**Giacomo Grassi**, *User Experience & Digital Process Director* presso INPS, ha descritto la sua esperienza di creazione di un design system nel settore pubblico. Ha quindi spiegato le specifiche necessità per un ente pubblico di grande complessità, che offre oltre 450 servizi web. Ha sottolineato l’importanza di creare team di lavoro coesi, valorizzare le competenze interne preesistenti e coinvolgere anche i collaboratori esterni fin dal primo giorno di lavoro.

Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.

Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali.

**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e [scarica la presentazione](/assets/images/posts/2022-02-22/Importanza-di-un design-system-per-i-servizi-digitali.pdf).

> Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).

Reviewer: ```suggestion
Per saperne di più **resta in contatto su [Slack](https://slack.developers.italia.it)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```

> Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).

Reviewer: ```suggestion
Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```

---
layout: post
title: "Come si realizza un design system per i servizi pubblici, parola agli esperti"
subtitle: Pubblicato il video della Community call del 9 febbraio sull’importanza del design system
date: 2022-02-22 12:00:00 +0200
tags: developers designers designsystem
categories: news
author: Michela Gabrieli
image: /assets/icons/logo-it.png
locale: it
---

> Grande successo per la Community call dello **scorso 9 febbraio**, dedicata al *design system* per i servizi pubblici digitali, con oltre **150 partecipanti**. Abbiamo parlato dell’importanza del *design system** per creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. Per farlo ci siamo affidati ad alcuni dei protagonisti di questo processo: Daniele Tabellini, Marco Liberati e Giacomo Grassi.

Reviewer: ```suggestion
Grande successo per la Community call dello **scorso 9 febbraio**, dedicata al *design system* per i servizi pubblici digitali, con oltre **150 partecipanti**. Abbiamo parlato dell’importanza del *design system* per creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. Per farlo ci siamo affidati ad alcuni dei protagonisti di questo processo: Daniele Tabellini, Marco Liberati e Giacomo Grassi.
```

**Marco Liberati**, membro attivo della community, è poi intervenuto per raccontare lo stato del Kit React “design-react-kit”, che supporta chi sviluppa nell’implementazione di Bootstrap Italia. Marco ha prima illustrato le basi del kit e ne ha poi raccontato l’evoluzione delle varie versioni fino all’attuale 4.0.

**Giacomo Grassi**, *User Experience & Digital Process Director* presso INPS, ha descritto la sua esperienza di creazione di un design system nel settore pubblico. Ha quindi spiegato le specifiche necessità per un ente pubblico di grande complessità, che offre oltre 450 servizi web. Ha sottolineato l’importanza di creare team di lavoro coesi, valorizzare le competenze interne preesistenti e coinvolgere anche i collaboratori esterni fin dal primo giorno di lavoro.

Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.

Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali.

> 

Reviewer: ```suggestion
```

**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e [scarica la presentazione](/assets/images/posts/2022-02-22/Importanza-di-un design-system-per-i-servizi-digitali.pdf).

Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).

> 
> 

Reviewer: ```suggestion
```

> Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) e [Designers Italia](https://twitter.com/developersITA?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor).

Reviewer: ```suggestion
Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).
```
